app/views/groups/select_suite.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Select a Suite</h1>
+<%= simple_form_for [@draw, @group], url: assign_suite_draw_group_path(@draw, @group) do |f| %>
+  <%= f.input :suite, collection: @suites, as: :radio_buttons, label_method: :number %>
+  <%= f.submit "Submit Selection" %>
```

Comment:
Do we want to add a tooltip here? Maybe something about how this is final and commits them to living on campus?